packages/solid/.storybook/preview-head.html

```diff
@@ -21,11 +21,11 @@
   /* transition background from Story to Doc */
   .sb-preparing-docs,
   .sb-preparing-story {
-    background-color: #1b1c1d;
+    background-color: #8D8F90;
```

Comment:
lets add a custom property
```
  body {
    --preview-background-color: #333333;
  }
```

and use that for this color. its a little greyer than our existing background but not quite as light as the proposed one